packages/kbn-io-ts-utils/src/array_to_string_rt/index.test.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import * as rt from 'io-ts';
+import { arrayToStringRt } from '.';
+import { isRight, Either, isLeft, fold } from 'fp-ts/lib/Either';
+import { identity } from 'fp-ts/lib/function';
+
+function getValueOrThrow(either: Either<unknown, any>) {
+  return fold(() => {
+    throw new Error('Cannot get right value of left');
+  }, identity)(either);
+}
+
+describe('arrayToStringRt', () => {
+  it('should validate strings', () => {
+    expect(isRight(arrayToStringRt.decode(''))).toBe(true);
+    expect(isRight(arrayToStringRt.decode('message'))).toBe(true);
+    expect(isRight(arrayToStringRt.decode('message,event.original'))).toBe(true);
+    expect(isLeft(arrayToStringRt.decode({}))).toBe(true);
+    expect(isLeft(arrayToStringRt.decode(true))).toBe(true);
```

Comment:
Nit: Could we keep using `isRight` in all tests so that the `toBe` part is what changes (I think this is more common and might make it easier to read the test at a glance).

Reply:
Sure, I don't have a strong preference and the suggestion makes sense, thank you! Fixed in 
[refactor(io-ts-utils): consistently use isRight for assertions](https://github.com/elastic/kibana/pull/183806/commits/65f00eac0793dc5afc8efe0ef3ca24eff3172ef0)